app/src/androidTest/java/org/mozilla/fenix/ui/robots/SearchRobot.kt

```diff
@@ -169,11 +187,14 @@ class SearchRobot {
         fun submitQuery(query: String, interact: BrowserRobot.() -> Unit): BrowserRobot.Transition {
             sessionLoadedIdlingResource = SessionLoadedIdlingResource()
             mDevice.waitForIdle()
-            browserToolbarEditView().perform(typeText(query + "\n"))
+            browserToolbarEditView().setText(query + "\n")
 
             runWithIdleRes(sessionLoadedIdlingResource) {
-                onView(ViewMatchers.withResourceName("browserLayout"))
-                    .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
+                assertTrue(
```

Comment:
I think this is one of the asserTrue that is failing, can't we just leave the 
```
mDevice.findObject(
                        UiSelector().resourceId("$packageName:id/browserLayout")
                    ).waitForExists(waitingTime)
```
part?

Reply:
let's see once the new test run finishes